recipes/tbb/oneTBB/conanfile.py

```diff
@@ -0,0 +1,115 @@
+from conan.tools.cmake import CMake, CMakeToolchain
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os, re
```

Comment:
Please one line per import (and alphabetical order).
```suggestion
import os
import re
```